samples/orleans/OrleansServer/Program.cs

```diff
@@ -0,0 +1,49 @@
+using Orleans.Runtime;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+builder.AddKeyedAzureTableService("clustering");
+builder.AddKeyedAzureBlobService("grainstate");
+builder.UseOrleans();
```

Comment:
I pulled the branch and this method doesn't exist? Where is it coming from?

Reply:
Fixed in latest